lms/static/js/jquery.gradebook.js

```diff
@@ -57,16 +58,23 @@ var Gradebook = function($element) {
 	};
 
 	var updateWidths = function(e) {
+		// Here updating direction of scroll.
 		maxScroll = $gradeTable.width() - $grades.width();
-		var targetLeft = clamp($gradeTable.position().left, -maxScroll, 0);
+		var targetLeft = clamp($gradeTable.position().left, maxScroll, 0);
 		updateHorizontalPosition(targetLeft);
 		setShadows(targetLeft);
 	};
 
+	/**
+	 * On dragging we compute pixels on the base of how mouse movement.
+	 * Then we are using those pixels to scroll left or right.
+	 * (Note: In past grade book was using dragging to
+	 * view hidden columns on viewport).
+	 *
+	 * @param left - Pixels to scroll left, When 0 it is on left most.
```

Comment:
This is confusing. I *think* you mean:
```
@param left: the number of pixels to move the gradebook to the left. If the value is 0,
             the gradebook should be moved all the way to the left side of the screen.
```
Is that correct? I don't see any special handling of the `0` case, though...